pgrx/src/datum/into.rs

```diff
@@ -337,7 +337,7 @@ impl_into_datum_c_str!(&CStr);
 impl<'a> IntoDatum for &'a [u8] {
     /// # Panics
     ///
-    /// This function will panic if the string being converted to a datum is longer than a `u32`.
+    /// This function will panic if the string being converted to a datum is longer than a half of [`i32::MAX`].
```

Comment:
Let's be precise: greater than or equal to 1GB *after* adding the varlena header (which is currently 4 bytes).

Reply:
Used `GiB` to be more precise since.